28 modelagem de dados

antoniotoineto opened the pull request:
## _Issue_ relacionada

Issue #28 

<!-- Este projeto só aceita pull request de issues abertas -->
<!--  Se você está propondo um novo recurso ou modificação, primeiro discuta-o com uma issue --> 
<!-- A issue deve ser referenciada diretamente com #codigo da issue -->

## Descrição

Foi realizado a Modelagem de Dados do projeto, fornecendo 3 versões diferentes. Cabe ao time e aos validadores junto conosco definir a melhor versão.
As três versões estão na pasta docs->modelagemDados
E também já está no mkdocs.yml

<!-- Descreva suas alterações em detalhes -->
<!-- Quem for revisar o pull request deve conseguir entender, apenas com esta, o que são as alterações feitas e como verificar seu funcionamento -->
## Imagens:
Versão 1:

![Group 8](https://github.com/fga-eps-mds/2023.1-Dnit-DOC/assets/54555684/b4491bcb-6307-4814-817d-1784c78c36cc)

Versão 2:
![Group 9](https://github.com/fga-eps-mds/2023.1-Dnit-DOC/assets/54555684/553aaef1-ef13-445f-aae4-b3f0daa69697)

Versão 3:
![Group 6](https://github.com/fga-eps-mds/2023.1-Dnit-DOC/assets/54555684/414e6cd7-2d27-4a07-b681-4beca6c95339)

<!-- Adicione imagens que demonstrem  a modificação em funcionamento -->
## Tipo de alterações
<!-- Quais tipos de alterações foram feitas -->
- [ ] _correção_ _de_ _bug_
- [ ] Nova _feature_
- [x] Documentação

## Lista de controle:

- [x] Atribui este pull request ao projeto
- [ ] Meu código segue a folha de estilos implementada
- [x] Meus _commits_ seguem a política de commits do projeto.
- [x] Minhas alterações requerem modificações na documentação atual.
- [x] Atualizei a Documentação
- [ ] Adicionei testes para cobrir minhas alterações.
- [ ] Todos os testes foram aprovados.

antoniotoineto committed:
Adicionando Modelagem de Dados

Co-authored-by: João Gabriel de Matos <joaogabriel.c.matos@gmail.com>

antoniotoineto committed:
#28 Adicionando Modelagem de dados no mkdocs.yml

antoniotoineto committed:
#28 Retirando documentos desnecessários

isadoragalvaoss commented:
Em relação a matrícula do funcionário DNIT (na versão 3) , na última reunião foi comentado que nem todos os funcionários tem essa matrícula. Devemos prosseguir com isso?

antoniotoineto commented:
Boa colocação @isadoragalvaoss . 
Sobre esse atributo matrícula,  mantemos ele apenas na entidade especializada "Funcionário DNIT" pois estamos assumindo que, se o funcionário for do DNIT, então ele terá matrícula. Perceba que na entidade "Funcionário Terceirizada" não temos o atributo Matrícula. Faz sentido para você seguir dessa forma?

O @JongaMatos vai colocar mais algumas coisas a mais e, se estiver tudo ok por vocês, podemos merjar

LeticiaResende23 committed:
Adicionado o versionamento

ArturRSA19 committed:
Alteração dos protótipos de alta fidelidade

ArturRSA19 committed:
Alteração dos protótipos de alta fidelidade
Co-authored-by: Artur Rodrigues <artur190503@outlook.com>
Co-authored-by: Letícia Resende <letigy78@gmail.com>

ArturRSA19 committed:
Alteração nos protótipos de alta

Co-authored-by: Letícia Resende <letigy78@gmail.com>

DenysRogeres commented:
Fiquei com a seguinte dúvida, a parte de matrícula deve ser removido da modelagem, uma vez que iremos utilizar apenas a UF. Além disso, UF seria um atributo do funcionário DNIT, funcionário de empresa terceirizada também teria esse atributo? 


Outro ponto: acredito que quando criamos essa issue foi pensando na modelagem do usuário pra implementação do cadastro e do login, mas vocês entregaram uma visão macro é isso já agregou muito. No entanto eu acho que seria importante ter uma visão do DER de usuário para podemos passarmos isso para o banco de dados

antoniotoineto commented:
Legal @DenysRogeres, então acho que podemos remover o atributo matrícula, já que não vai agregar em muito e o usuário já disse que poderia remover. Mantive UF do funcionário apenas em Func. DNIT.
Sobre o DER específico do usuário vamos alinhar como podemos fazer

antoniotoineto commented:
## Nova versão proposta:

![Group 6 (1)](https://github.com/fga-eps-mds/2023.1-Dnit-DOC/assets/54555684/af2cfcea-7502-44d7-a7a3-951169efaddf)

yandrade1305 committed:
#41 Adicionando observação

yandrade1305 commented:
Eu vi que visita tá com dois atributos status, por quê?
Fora isso está tudo certo

JongaMatos committed:
#22 Adicionando manual da marca

JongaMatos committed:
#22 Adicionando manual da marca

JongaMatos committed:
#22 Arrumando estetica da pagina

JongaMatos committed:
#22 Arrumando estetica da pagina

JongaMatos committed:
Revert "#41 Adicionando observação"

JongaMatos committed:
#46 Adicionando workflow para verificar se o build é possivel

JongaMatos committed:
#46 Fazendo tentativa de ajuste na chamada de build.

JongaMatos committed:
#46 Alterando chamada para modo não verboso para facilitar identificação de erro.

JongaMatos committed:
#46 Adicionando comanando de instalação do tema.

JongaMatos committed:
Correção de warning

JongaMatos committed:
#46 Adicionando workflow de deploy.

#46 corrigindo indentação dploy.yml

#46 tentando corrigir sitaxe yml.

#46 tentando corrigir sitaxe yml.

JongaMatos committed:
Corrigindo evento do github actions para o deploy

JongaMatos committed:
Update deploy.yml

JongaMatos committed:
#28 Finalizando descrição do DER.

JongaMatos committed:
Merge branch 'main' into 28-ModelagemDeDados

JongaMatos commented:
@yandrade1305 pode revisar?

antoniotoineto committed:
Adicionando MER

lucas229 committed:
Criação do documento de arquitetura #25

lucas229 committed:
Adicionando visão de dados ao doc de arquitetura #25

lucas229 committed:
Merge branch 'main' into 28-ModelagemDeDados

lucas229 reviewed (approved):
Adicionei a modelagem ao documento de arquitetura.
